src/frontend/src/CustomNodes/GenericNode/components/NodeDescription/index.tsx

```diff
@@ -115,7 +114,7 @@ export default function NodeDescription({
           {charLimit && (
             <div
               className={cn(
-                "text-left text-xs",
+                "text-left text-[13px]",
```

Comment:
These text elements appear to be 14px in figma

Reply:
![image](https://github.com/user-attachments/assets/836bbd7f-6779-4d3e-88e1-e1bd67cf6524)

thats strange I have 13px on mine...